src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1436,6 +1443,12 @@ void Convolution::prepareParams() {
         primArgs[DNNL_ARG_WEIGHTS] = wghMemPtr->GetPrimitive();
         primArgs[DNNL_ARG_DST] = dstMemPtr->GetPrimitive();
 
+        if (key.constWeight) {
+            if (!prevExecPtr || prevExecPtr->getWeightDesc() != execPtr->getWeightDesc()) {
+                primArgs[DNNL_ARG_WEIGHTS] = prepareWeightMemory(DnnlExtensionUtils::makeDescriptor(execPtr->getWeightDesc()))->GetPrimitive();
+            }
```

Comment:
Is not it more reliable to do the following:
```suggestion
            auto itr = primArgs.find(DNNL_ARG_WEIGHTS);
            if (itr == primArgs.end() || itr.second.get_desc() != execPtr->getWeightDesc()) {
                primArgs[DNNL_ARG_WEIGHTS] = prepareWeightMemory(DnnlExtensionUtils::makeDescriptor(execPtr->getWeightDesc()))->GetPrimitive();
            }
```
Also that would be great to extract this procedure to some common part as well, simply to reduce the number of possible errors in implementation.